### ISSUE TITLE
Sample Implementation 

### ISSUE THREAD

**Issue by saadqbal** [skrafft/react-native-jitsi-meet#10]
Hi @skrafft 

Great work. Can this library be used with custom implentation of Jitsi? Could you please show a sample app?

**Comment by skrafft** [comment]
Hi @saadqbal,

It definitely can be used with a custom implementation of Jitsi. That's what we do at my company. If you want to have a preview of it, you have to register at [https://www.ibani.com](https://www.ibani.com). The onboarding process uses the video platform.

**Comment by saadqbal** [comment]
Hi @skrafft ,
Thanks for the reponse. No doubt Iban is a great initiative. best for luck for your startup.

I am unable to find the `libRNJitsiMeet.a` file. Could you please let me know where can I find this file?

**Comment by skrafft** [comment]
Hi,

libRNJitsiMeet.a is generated when compiling react-native-jitsi-meet iOS project. Can you check that followed all the install instructions ?

Regards.

**Comment by saadqbal** [comment]
Yup I have followed the instructions. I asked the libRNJitsiMeet.a because I am getting the following error:
`Unknown class JitsiMeetView in Interface Builder file.`

Below is the complete log.

```
2018-11-11 23:40:02.384108+0500 videocall[43454:2547840] Running application "videocall" with appParams: {"rootTag":1,"initialProps":{}}. __DEV__ === true, development-level warning are ON, performance optimizations are OFF
2018-11-11 23:40:02.401 [warn][tid:com.facebook.react.JavaScript] Jitsi
2018-11-11 23:40:02.401358+0500 videocall[43454:2547840] Jitsi
2018-11-11 23:40:02.405 [info][tid:com.facebook.react.RNJitsiMeetNavigatorManagerQueue][RNJitsiMeetNavigator.m:49] Initialize
2018-11-11 23:40:02.405364+0500 videocall[43454:2547844] Initialize
2018-11-11 23:40:03.412 [info][tid:com.facebook.react.RNJitsiMeetNavigatorManagerQueue][RNJitsiMeetNavigator.m:56] Load URL https://video.[maskedUrl].com/sengaytest
2018-11-11 23:40:03.412431+0500 videocall[43454:2547825] Load URL https://video.sengay.com/sengaytest
2018-11-11 23:40:03.415332+0500 videocall[43454:2547749] Unknown class JitsiMeetView in Interface Builder file.
2018-11-11 23:40:03.416448+0500 videocall[43454:2547749] -[UIView setDelegate:]: unrecognized selector sent to instance 0x7fea80d055c0
2018-11-11 23:40:03.419685+0500 videocall[43454:2547749] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UIView setDelegate:]: unrecognized selector sent to instance 0x7fea80d055c0'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000110c051e6 __exceptionPreprocess + 294
	1   libobjc.A.dylib                     0x000000010fb38031 objc_exception_throw + 48
	2   CoreFoundation                      0x0000000110c86784 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   UIKit                               0x0000000112ee96db -[UIResponder doesNotRecognizeSelector:] + 295
	4   CoreFoundation                      0x0000000110b87898 ___forwarding___ + 1432
	5   CoreFoundation                      0x0000000110b87278 _CF_forwarding_prep_0 + 120
	6   videocall                           0x000000010c54c788 -[JitsiMeetViewController setDelegate:] + 104
	7   videocall                           0x000000010c54d13f __36-[RNJitsiMeetNavigatorManager call:]_block_invoke + 287
	8   libdispatch.dylib                   0x00000001159187ec _dispatch_client_callout + 8
	9   libdispatch.dylib                   0x0000000115925642 _dispatch_sync_thread_bound_invoke + 386
	10  libdispatch.dylib                   0x00000001159187ec _dispatch_client_callout + 8
	11  libdispatch.dylib                   0x00000001159238cf _dispatch_main_queue_callback_4CF + 628
	12  CoreFoundation                      0x0000000110bc7c99 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	13  CoreFoundation                      0x0000000110b8bea6 __CFRunLoopRun + 2342
	14  CoreFoundation                      0x0000000110b8b30b CFRunLoopRunSpecific + 635
	15  GraphicsServices                    0x0000000118c75a73 GSEventRunModal + 62
	16  UIKit                               0x0000000112cbb057 UIApplicationMain + 159
	17  videocall                           0x000000010c54c6bf main + 111
	18  libdyld.dylib                       0x0000000115995955 start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException

```

**Comment by skrafft** [comment]
Hi, it seems you missed a step in the manual install. Please make sure that you followed steps 2 to 4 in the install guide.

**Comment by pohsiu** [comment]
@skrafft I have same issue, I double check step 2 to 4, seems like all set up fine except one step that I am not sure which is set right 4-6.) Add the following script in a new "Run Script" phase in "Build Phases":,
I tried New run script and replace original code of Bundle React... place, 
Both of these won't work ...
Does it have any other possible?

**Comment by saadqbal** [comment]
Hi @skrafft ,
Are you availabel or a paid task related to related to react native jisti-meet? 
Please email me your skype or other id at hiappstudio@gmail.com.

**Comment by skrafft** [comment]
@saadqbal, @pohsiu , I just published v1.0.6 that should fix the problem
